File: restapi/doctorconsultantrestapi/src/main/java/com/nitw/doctorconsultantrestapi/models/LoginData.java
package com.nitw.doctorconsultantrestapi.models;

import java.util.Map;

public class LoginData{
	String userName;
	String password;
	
	public LoginData() {}
	public LoginData(Map<String, String> loginData){
		this.userName = loginData.get("userName");
		this.password = loginData.get("password");
	}
	public String getUserName(){
        return this.userName;
    }
	public String getPassword(){
        return this.password;
    }
	public void setUserName(String userName){
        this.userName = userName;
    }
	public void setPassword(String password){
        this.password = password;
    } 
}

File: AngularApp/src/app/login-data.ts
export class LoginData {
    userName:String;
    password:String;
    getUserName(){
        return this.userName;
    }
    getPassword(){
        return this.password;
    }
    setUserName(userName:String){
        this.userName = userName;
    }
    setPassword(password:String){
        this.password = password;
    }   
}
